CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [CHANGE] All: set `-server.report-grpc-codes-in-instrumentation-label-enabled` to `true` by default, which enables reporting gRPC status codes as `status_code` labels in the `cortex_request_duration_seconds` metric. #7144
 * [CHANGE] Distributor: report gRPC status codes as `status_code` labels in the `cortex_ingester_client_request_duration_seconds` metric by default. #7144
 * [CHANGE] Distributor: CLI flag `-ingester.client.report-grpc-codes-in-instrumentation-label-enabled` has been deprecated, and its default value is set to `true`. #7144
+* [CHANGE] Ingester: CLI flag `-ingester.return-only-grpc-errors` has been deprecated, and its default value is set to `true`. #7151
```

Comment:
We should probably note that if someone is upgrading from Mimir 2.10 or earlier, they need to set this to `false` during upgrade. Once all components are on 2.11 or later, it's safe to switch to default. (This will need to be part of release notes for 2.12 too)

(please double-check if what I wrote is 100% correct.)

Reply:
I have updated the comment.